src/yunohost/log.py

```diff
@@ -309,6 +335,8 @@ def __init__(self, operation, related_to=None, **kwargs):
         self.ended_at = None
         self.logger = None
         self._name = None
+        self.data_to_redact = []
+        self.data_to_redact.append(open("/etc/yunohost/mysql", "r").read().strip())
```

Comment:
What about to add also the postgresql password in `/etc/yunohost/psql` if it exist ?

Reply:
Should be done via the lasts commits :+1: